### ISSUE TITLE
Thumbnails not create for some jpgs

### ISSUE THREAD

**Issue by HurstConsulting** [trippo/ResponsiveFilemanager#289]
I have been tested with a number of jpg images and thumbnails are not successfully created for many of them. The images get uploaded but no thumbnail file exists. However, some jogs work fine so I am not sure what the issue is.

**Comment by Elendurwen** [comment]
I'd be interested in knowing about this as well.

**Comment by trippo** [comment]
It's possible that the jpg is too big for server memory

**Comment by Elendurwen** [comment]
No, I don't think so - I tried uploading a 130kb jpeg with no thumbnail. I uploaded larger jpegs and thumbnails were ok.

**Comment by trippo** [comment]
The same jpeg don't work also in demo site?

**Comment by Elendurwen** [comment]
Hmm it does work on the demo site. Any idea why this would be? I have specified the thumbs folder correctly (as some thumbnails do work) and I have done chmod 755 on them (I need to 755 rather than 766 as suggested due to my server settings so that I can see any files at all).

**Comment by trippo** [comment]
I know new version fix this and return to 755

**Comment by Elendurwen** [comment]
Sorry not sure what you mean - I downloaded the latest version from the releases page and set the thumbnails folder to chmod 755 and it doesn't work.

**Comment by Elendurwen** [comment]
Also - I just tried different chmods on my uploads and upload_thumbs directories - doesn't matter if I use 755, 766 or 777, thumbnails on some images are still not created. Could this have anything to do with the server's setup? Maybe some packages used for resizing images etc?

**Comment by trippo** [comment]
GD library

**Comment by Elendurwen** [comment]
Which version of the GD library do you use on your server?

**Comment by trippo** [comment]
2.4.9

**Comment by adc-diegocam** [comment]
Raise your max file size.. or your PHP's memory limit. Make sure your PHP memory limit is set in Megabytes (i,e, memory_limit = 1000M) . Do not use "G" as gigabytes, their code does not support it.
